packages/hackathon/package.json

```diff
@@ -29,43 +29,44 @@
   ],
   "scripts": {
     "analyze": "export ANALYZE_MODE=static && yarn bundle",
-    "bundle": "tsc && webpack --config webpack.prod.config.js",
+    "bundle": " tsc && NODE_OPTIONS=--openssl-legacy-provider webpack --config webpack.prod.config.js",
```

Comment:
Do we need this for the same extensions as well?

Reply:
updated